compiler/noirc_frontend/src/hir/resolution/resolver.rs

```diff
@@ -537,6 +537,7 @@ impl<'a> Resolver<'a> {
 
             let result = self.interner.get_type_alias(id).get_type(&args);
 
+            self.interner.add_type_alias_ref(id, Location::new(span, self.file));
```

Comment:
Could add a comment here as well since this is only needed for the LSP.

Reply:
f341469: chore: review nits